### ISSUE TITLE
Room member details: Add mute option in ADMIN TOOLS section

### ISSUE THREAD

**Issue by giomfo** [vector-im/vector-android#652]
The room admin should be able to mute a room member. This action is performed by updating the member power level with -1 value.

**Comment by t3chguy** [comment]
This is not correct, its the user default (from room state `m.room.power_levels`) -1 to Mute, because the default itself could be -ve
